tests/src/test_vespa.py

```diff
@@ -21,6 +22,40 @@
 from tests.conftest import get_parser_output, FIXTURE_DIR
 
 
+@pytest.mark.parametrize(
+    ("metadata", "expected"),
+    [
+        ({}, []),
+        (
+            {"topic": ["Adaptation"]},
+            [VespaFamilyDocument.MetadataItem(name="topic", value="Adaptation")],
+        ),
+        (
+            {
+                "hazard": [],
+                "sector": [
+                    "Adaptation",
+                    "Economy-wide",
+                    "Waste",
+                    "Agriculture",
+                    "Water",
+                ],
+            },
+            [
+                VespaFamilyDocument.MetadataItem(name="sector", value="Adaptation"),
+                VespaFamilyDocument.MetadataItem(name="sector", value="Economy-wide"),
+                VespaFamilyDocument.MetadataItem(name="sector", value="Waste"),
+                VespaFamilyDocument.MetadataItem(name="sector", value="Agriculture"),
+                VespaFamilyDocument.MetadataItem(name="sector", value="Water"),
+            ],
+        ),
```

Comment:
Related to the suggestion above.
```suggestion
        ),
        (None, None),
```

Reply:
👍